test/unit/core/get.spec.js

```diff
@@ -619,130 +619,118 @@ describe('Get', function() {
-                                                name: 'ids'
-                                            },
-                                            'rating'
-                                        ],
-                                        value: {
-                                            path: ['videos', 123, 'rating'],
-                                            value: 'rating 123'
-                                        }
-                                    }
+                                route: 'videos[{integers:ids}].title',
+                                start: 1,
+                                end: 2,
+                                responses: [
+                                    {path: ['video', 1, 'title'], value: 'Orange Is The New Black'},
+                                    {path: ['video', 2, 'title'], value: 'Whatever'}
                                 ],
-                                start: 1
-                            },
-                            {
-                                path: ['videos', 124, ['title', 'rating']],
-                                routes: [
-                                    {
-                                    end: 5,
-                                    requested: [ 'videos', [124], 'title' ],
-                                    virtual: [
-                                        'videos',
-                                        {
-                                            type: '\u001eintegers',
-                                            named: true,
-                                            name: 'ids'
-                                        },
-                                        'title'
-                                    ],
-                                    value: {
-                                        path: ['videos', 124, 'title'],
-                                        value: 'title 124'
-                                    }
-                                    },
-                                    {
-                                    end: 6,
-                                    requested: ['videos', [124], 'rating'],
-                                    virtual: [
-                                        'videos',
-                                        {
-                                            type: '\u001eintegers',
-                                            named: true,
-                                            name: 'ids'
-                                        },
-                                        'rating'
-                                    ],
-                                    value: {
-                                        path: ['videos', 124, 'rating'],
-                                        value: 'rating 124'
-                                    }
+                                paths: ['videos', [1, 2], 'title']
+                            }
```

Comment:
Typo. You put "video" instead of "videos" above but put "videos" in the path. That's why you end up with two atoms in the responses. Please fix.